<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #000000;
    font-family: 'Helvetica Neue', sans-serif;
}

QLabel#label {
    background-color: #000000;
    color: #ffffff;
    border: none;
    padding: 20px 10px;
    font-size: 48px;
    font-weight: 300;
    text-align: right;
    margin-bottom: 10px;
    text-align: right;  
    padding-right: 15px;  
qproperty-alignment: 'AlignRight';
max-width: 100%;
    min-width: 350px;
    text-overflow: ellipsis;
    overflow: hidden;
}

QPushButton {
    border-radius: 40px;
    min-width: 80px;
    min-height: 80px;
    font-size: 32px;
    border: none;
    margin: 5px;
    transition: background-color 0.2s; 
}

#num0, #num1, #num2, #num3, #num4,
#num5, #num6, #num7, #num8, #num9,
#dot {
    background-color: #333333;
    color: white;
}

#opPlus, #opMinus, #opMultiply, #opDivide,
#opEqual {
    background-color: #ff9f0a;
    color: white;
}

#opClear, #opClearOne, #opExp, #opSqrt, #opLparen, #opRparen, #plusminus {
    background-color: #a5a5a5;
    color: black;
    font-size: 28px;
}

QPushButton:pressed {
    padding-top: 2px;
    padding-left: 2px;
}

#num0:pressed, #num1:pressed, #num2:pressed, #num3:pressed,
#num4:pressed, #num5:pressed, #num6:pressed, #num7:pressed,
#num8:pressed, #num9:pressed, #dot:pressed {
    background-color: #4d4d4d !important;
}

#opPlus:pressed, #opMinus:pressed,
#opMultiply:pressed, #opDivide:pressed,
#opEqual:pressed {
    background-color: #ffb733 !important;
}

#opClear:pressed, #opClearOne:pressed, #opExp:pressed,
#opSqrt:pressed, #opLparen:pressed,
#opRparen:pressed, #plusminus:pressed {
    background-color: #c0c0c0 !important;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="1">
       <widget class="QPushButton" name="num2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="num7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="opMinus">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="opMultiply">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="num5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="opPlus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="num3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="num4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="dot">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="num9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="opSqrt">
        <property name="text">
         <string>âˆš</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="num8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="opClear">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="plusminus">
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="opExp">
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="opLparen">
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="num6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="num0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="num1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="opRparen">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="opClearOne">
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="opDivide">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="opEqual">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
